Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs

```diff
@@ -5045,7 +5045,7 @@ protected override void UpdateTractiveForce(float elapsedClockSeconds, float t,
                     // Crank Angle - converts the above range to 0 - 180 - 0 - this is the principle reference used so that it lines up with reference
                     // tables used to buold this function
                     // Normalised Crank Angle - converts the above to a 0 - 360 range, this is used for triggering special steam effects, etc.
-                    float axlePostionRad = LocomotiveAxles[0].AxlePositionRad;
+                    float axlePostionRad = (float)LocomotiveAxles[0].AxlePositionRad;
```

Comment:
I don't understand this typo. It is a temporary parameter only used in the following line, and was named slightly differently to differentiate it from the "real" parameter.

If you wish me to change it then can you explain in a bit more detail.

Thanks

Reply:
Ah, so using "postion" instead of "Position" is intentional? I didn't know that intentionally adding typos is a coding practice. I assume it is OK then, although I'd prefer using descriptive names for variables instead of adding typos. That's a personal preference though.

Reply:
Ok, I have renamed it for clarity.